### PR TITLE
RSWEB-8024: refactor valueProps pattern

### PR DESCRIPTION
**Pull request by writes**
[zoo]
https://jira.rax.io/browse/RSWEB-8024 - https://github.com/rackerlabs/zoolander/pull/343

http://local.rackspace.com:7080/managed-hosting

[www]
https://jira.rax.io/browse/RSWEB-8029 - https://github.rackspace.com/rswebteam/www/pull/1533

http://localhost:9000/derek/solutions/solutions-usage

### CODE REVIEW

**Comment by WidgetsBurritos** [comment]
👍 but definitely needs www counterpart. If we get www in sync with this, the alignment issues should get fixed.

**Comment by WidgetsBurritos** [comment]
👍

**Comment by brantdarilek** [comment]
👍

File: styleguide/derek/layouts/value-props.ejs [hunk 1]
```diff
@@ -10,7 +10,7 @@
       }) %>
     </div>
```
Comment (amberRrucker) [inline styleguide/derek/layouts/value-props.ejs]: Should we change the name of these files to reflect the valueProp name change?

**Comment by amberRrucker** [comment]
Value Prop pattern should actually not contain a link. The EBPs are aware and there should be no value props on the site utilizing a link or button. Those should be all i&S if they have a link.

screen shot 2017-01-10 at 10 31 56 am
screen shot 2017-01-10 at 10 32 12 am

File: styleguide/derek/layouts/value-props.ejs (hunk 1 above)
Comment (WidgetsBurritos) [inline styleguide/derek/layouts/value-props.ejs]: Probably not be a bad idea.

**Comment by writes** [comment]
Hey @amberRrucker thank you! I have made the updates requested. Let me know if you see anything else.

**Comment by amberRrucker** [comment]
+1 thanks JT.

**Pull request merged**